### PR TITLE
Fixes SDQL for (hopefully) the last time by removing try catches. Removes automatic runtime tracking from SDQL.

### LINKED ISSUE #29500: Runtimes in sdql2 proc calls breaks all admin proc calls.
because sdql2 abuses try/catch, the wrapper never finishes if a runtime happens since the try/catch unwinds the stack.

@kevinz000

### PR DESCRIPTION
**Pull request by silicons**
Removes try/catches.
fixes #29500

### CODE REVIEW

File: code/modules/admin/verbs/SDQL2/SDQL_2.dm [hunk 1]
```diff
@@ -76,99 +73,79 @@
...
+		for(var/v in sets)
+			if(++i == sets.len)
+				temp.vv_edit_var(v, SDQL_expression(d, set_list[sets]))
+				break
+			if(temp.vars.Find(v) && (istype(temp.vars[v], /datum)))
+				temp = temp.vars[v]
+			else
+				break
 
 /proc/SDQL_parse_exception(exception/E)
```
Comment (RemieRichards) [inline code/modules/admin/verbs/SDQL2/SDQL_2.dm]: this is now unused.

File: code/modules/admin/verbs/SDQL2/SDQL_2.dm [hunk 2]
```diff
@@ -76,106 +73,79 @@
...
-
-/proc/SDQL_parse_exception(exception/E)
-	var/list/returning = list()
-	returning += "Runtime Error: [E.name]<BR>"
-	returning += "Occured at line [E.line] file [E.file]<BR>"
-	returning += "Description: [E.desc]<BR>"
-	return returning
+
+/proc/SDQL_qdel_datum(datum/d)
+	qdel(d)
```
Comment (Cyberboss) [inline code/modules/admin/verbs/SDQL2/SDQL_2.dm]: Why

File: code/modules/admin/verbs/SDQL2/SDQL_2.dm [hunk 3]
```diff
@@ -76,106 +73,79 @@
...
+
+/proc/SDQL_gen_vv_href(t)
```
Comment (Cyberboss) [inline code/modules/admin/verbs/SDQL2/SDQL_2.dm]: I feel like this will only be exploited

File: code/modules/admin/verbs/SDQL2/SDQL_2.dm [hunk 4]
```diff
@@ -76,106 +73,79 @@
...
+	var/text = ""
+	text += "<A HREF='?_src_=vars;Vars=\ref[t]'>\ref[t]</A>"
+	if(istype(t, /atom))
+		var/atom/a = t
+		if(a.x)
+			text += ": [t] at ([a.x], [a.y], [a.z])<br>"
+
+		else if(a.loc && a.loc.x)
+			text += ": [t] in [a.loc] at ([a.loc.x], [a.loc.y], [a.loc.z])<br>"
```
Comment (Cyberboss) [inline code/modules/admin/verbs/SDQL2/SDQL_2.dm]: Coord macro

**Review by Cyberboss — DISMISSED** [review]

File: code/modules/admin/verbs/SDQL2/SDQL_2.dm (hunk 3 above)
Comment (MrStonedOne) [inline code/modules/admin/verbs/SDQL2/SDQL_2.dm]: hrefs are not a security boundary. it is trivial to guess hrefs

File: code/modules/admin/verbs/SDQL2/SDQL_2.dm (hunk 2 above)
Comment (MrStonedOne) [inline code/modules/admin/verbs/SDQL2/SDQL_2.dm]: likely to prevent the runtime from impacting the loop

**Comment by MrStonedOne** [comment]
`fixes #29500`

**Pull request merged**